mods/common/languages/en.ftl

```diff
@@ -117,469 +119,468 @@ overwrite-map-outside-edit-prompt = "The map has been edited from outside the ed
 overwrite-map-outside-edit-confirm = Save
 
 ## GameInfoLogic
-objectives = Objectives
-briefing = Briefing
-options = Options
-debug = Debug
-chat = Chat
+label-objectives = Objectives
+label-briefing = Briefing
+label-options = Options
+label-debug = Debug
+label-chat = Chat
 
 ## GameInfoObjectivesLogic, GameInfoStatsLogic
-in-progress = In progress
-accomplished = Accomplished
-failed = Failed
+label-mission-in-progress = In progress
+label-mission-accomplished = Accomplished
+label-mission-failed = Failed
 
 ## GameInfoStatsLogic
-mute = Mute this player
-unmute = Unmute this player
+label-client-state-disconnected = Gone
+tooltip-mute-player = Mute this player
+tooltip-unmute-player = Unmute this player
 
-## GameInfoStatsLogic
-gone = Gone
-
-kick-title = Kick { $player }?
-kick-prompt = They will not be able to rejoin this game.
-kick-accept = Kick
+dialog-title-kick = Kick { $player }?
+dialog-prompt-kick = They will not be able to rejoin this game.
+dialog-accept-kick = Kick
 
 ## GameTimerLogic
-paused = Paused
-max-speed = Max Speed
-speed = { $percentage }% Speed
-complete = { $percentage }% complete
+label-paused = Paused
+label-max-speed = Max Speed
+label-percentage-speed = { $percentage }% Speed
+label-percentage-complete = { $percentage }% complete
```

Comment:
These both feel undescriptive to me. Perhaps`label-replay-speed` and `label-replay-completness`?

Reply:
Though probably it's better to just remove the word `percantage`